camayoc/tests/qcs/test_scan.py

```diff
@@ -31,55 +31,80 @@ def sources():
     return srcs
 
 
-def wait_until_complete(scan, timeout=120):
-    """Wait until the scan has either failed or completed.
+def first_source():
+    """Gather sources from config file."""
```

Comment:
I think this docstring can be expanded to mention that a default profile will be returned. Also the section in the qcs config should be named sources instead of profiles to align with the name on the system, what do you think?

Reply:
That's fine i just need to put a BIG STICKY NOTE on my monitor to update jenkins config file once this is merged